diff --git a/browser/branding/beta/configure.sh b/browser/branding/beta/configure.sh
index 0000000000..1111111111 100644
--- a/browser/branding/beta/configure.sh
+++ b/browser/branding/beta/configure.sh
@@ -3,4 +3,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+MOZ_APP_NAME=orbit-browser
 MOZ_APP_DISPLAYNAME="Orbit Browser Beta"
 
diff --git a/browser/branding/stable/configure.sh b/browser/branding/stable/configure.sh
index 0000000000..1111111111 100644
--- a/browser/branding/stable/configure.sh
+++ b/browser/branding/stable/configure.sh
@@ -3,4 +3,5 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+MOZ_APP_NAME=orbit-browser
 MOZ_APP_DISPLAYNAME="Orbit Browser"

