diff --git a/browser/branding/stable/content/jar.mn b/browser/branding/stable/content/jar.mn
index 490670db04..f53269a7ff 100644
--- a/browser/branding/stable/content/jar.mn
+++ b/browser/branding/stable/content/jar.mn
@@ -20,3 +20,4 @@ browser.jar:
   content/branding/icon64.png                    (../default64.png)
   content/branding/icon128.png                   (../default128.png)
   content/branding/aboutDialog.css
+  content/branding/orbit-newtab-bg.png
