diff --git a/browser/branding/beta/pref/firefox-branding.js b/browser/branding/beta/pref/firefox-branding.js
--- a/browser/branding/beta/pref/firefox-branding.js
+++ b/browser/branding/beta/pref/firefox-branding.js
@@ -3,9 +3,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-pref("startup.homepage_override_url", "https://zen-browser.app/whatsnew?v=%VERSION%");
-pref("startup.homepage_welcome_url", "https://zen-browser.app/welcome/");
-pref("startup.homepage_welcome_url.additional", "https://zen-browser.app/privacy-policy/");
+pref("startup.homepage_override_url", "");
+pref("startup.homepage_welcome_url", "https://safecircle.tech/orbit");
+pref("startup.homepage_welcome_url.additional", "https://safecircle.tech/privacy");
 
 // Give the user x seconds to react before showing the big UI. default=192 hours
 pref("app.update.promptWaitTime", 691200);
@@ -14,11 +14,11 @@
 // app.update.url.details: a default value for the "More information about this
 // update" link supplied in the "An update is available" page of the update
 // wizard.
-pref("app.update.url.manual", "https://zen-browser.app/download/");
-pref("app.update.url.details", "https://zen-browser.app/release-notes/latest/");
-pref("app.releaseNotesURL", "https://zen-browser.app/whatsnew/");
-pref("app.releaseNotesURL.aboutDialog", "https://www.zen-browser.app/release-notes/%VERSION%/");
-pref("app.releaseNotesURL.prompt", "https://zen-browser.app/release-notes/%VERSION%/");
+pref("app.update.url.manual", "https://github.com/safecircleia/orbit/releases");
+pref("app.update.url.details", "https://github.com/safecircleia/orbit/releases/latest");
+pref("app.releaseNotesURL", "https://github.com/safecircleia/orbit/releases/latest");
+pref("app.releaseNotesURL.aboutDialog", "https://github.com/safecircleia/orbit/releases/latest");
+pref("app.releaseNotesURL.prompt", "https://github.com/safecircleia/orbit/releases/latest");
 
 // Number of usages of the web console.
 // If this is less than 5, then pasting code into the web console is disabled
diff --git a/browser/branding/stable/pref/firefox-branding.js b/browser/branding/stable/pref/firefox-branding.js
--- a/browser/branding/stable/pref/firefox-branding.js
+++ b/browser/branding/stable/pref/firefox-branding.js
@@ -3,9 +3,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-pref("startup.homepage_override_url", "https://zen-browser.app/whatsnew?v=%VERSION%");
-pref("startup.homepage_welcome_url", "https://zen-browser.app/welcome/");
-pref("startup.homepage_welcome_url.additional", "https://zen-browser.app/privacy-policy/");
+pref("startup.homepage_override_url", "");
+pref("startup.homepage_welcome_url", "https://safecircle.tech/orbit");
+pref("startup.homepage_welcome_url.additional", "https://safecircle.tech/privacy");
 
 // Give the user x seconds to react before showing the big UI. default=192 hours
 pref("app.update.promptWaitTime", 691200);
@@ -14,11 +14,11 @@
 // app.update.url.details: a default value for the "More information about this
 // update" link supplied in the "An update is available" page of the update
 // wizard.
-pref("app.update.url.manual", "https://zen-browser.app/download/");
-pref("app.update.url.details", "https://zen-browser.app/release-notes/latest/");
-pref("app.releaseNotesURL", "https://zen-browser.app/whatsnew/");
-pref("app.releaseNotesURL.aboutDialog", "https://www.zen-browser.app/release-notes/%VERSION%/");
-pref("app.releaseNotesURL.prompt", "https://zen-browser.app/release-notes/%VERSION%/");
+pref("app.update.url.manual", "https://github.com/safecircleia/orbit/releases");
+pref("app.update.url.details", "https://github.com/safecircleia/orbit/releases/latest");
+pref("app.releaseNotesURL", "https://github.com/safecircleia/orbit/releases/latest");
+pref("app.releaseNotesURL.aboutDialog", "https://github.com/safecircleia/orbit/releases/latest");
+pref("app.releaseNotesURL.prompt", "https://github.com/safecircleia/orbit/releases/latest");
 
 // Number of usages of the web console.
 // If this is less than 5, then pasting code into the web console is disabled
